src/shell/commands/table_management.cpp

```diff
@@ -886,3 +888,106 @@ bool clear_app_envs(command_executor *e, shell_context *sc, arguments args)
     }
     return true;
 }
+
+bool get_max_replica_count(command_executor *e, shell_context *sc, arguments args)
+{
```

Comment:
`ls` command can show the max_replica_count, here add the command has some different?

Reply:
> `ls` command can show the max_replica_count, here add the command has some different?

Actually there is no special reason for adding `get_max_replica_count` currently. This interface is designed just for a complete get/set semantics.